tests/regression/roles/sonic_ntp/defaults/main.yml

```
@@ -28,11 +28,11 @@ preparations_tests:
     - "interface {{ vlan1 }}"
     - "interface {{ lo1 }}"
   delete_vrfs:
-    - "no ip vrf mgmt"
-    - "no ip vrf Vrf_1"
+    - "no ip vrf {{ vrf_2 }}"
+    - "no ip vrf {{ vrf_1 }}"
```

Please swap the order of these two lines.

```suggestion
    - "no ip vrf {{ vrf_1 }}"
    - "no ip vrf {{ vrf_2 }}"
```